.github/workflows/test.yml

```diff
@@ -36,3 +36,32 @@ jobs:
       run: |
         python -m coverage run --source packaging/ -m pytest --strict tests
         python -m coverage report -m --fail-under 100
+
+
+  test-pypy:
+    name: Test ${{ matrix.pypy_version }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        pypy_version: ['pypy2', 'pypy3']
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - uses: actions/setup-python@v1
+      name: Install Python ${{ matrix.pypy_version }}
+      with:
+        python-version: ${{ matrix.pypy_version }}
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install coverage pretend pytest==4.6.5
```

Comment:
Can you add a comment about the problem and a link to the pytest issue so we can unpin the version once this is fixed in a released version of pytest?

Reply:
Will do